src/google-classroom-extractor/google_classroom_extractor/mapping/user_submission_activities.py

```diff
@@ -117,6 +121,8 @@ def submissions_to_user_submission_activities_dfs(
                 "gradeChangeType",
                 "courseId",
                 "actorUserId",
+                "CreateDate",
+                "LastModifiedDate"
```

Comment:
You'll want to add empty "SourceCreateDate" and "SourceLastModifiedDate" fields as well.  See for example users.py lines 80-81

Reply:
✔️